tests/php81-compatibility.patch

```diff
@@ -374,3 +422,50 @@ index 6e5cca4..1d24bb6 100644
      {
      }
  
+@@ -23,7 +23,7 @@ class TrailingCommaOnFunctions
+     {
+         $var = null;
+ 
+-        $singleLine = static function (int $arg) use ($var): void {
++        $singleLine = static function (int $arg) use ($var,): void {
+             var_dump($var);
+         };
+ 
+@@ -37,7 +37,8 @@ class TrailingCommaOnFunctions
+ 
+ $class = new TrailingCommaOnFunctions();
+ 
+-$class->a(1);
++// phpcs:ignore Generic.Functions.FunctionCallArgumentSpacing.NoSpaceAfterComma
++$class->a(1,);
```

Comment:
Also a bad idea? Unless I'm misunderstanding the purpose of this file :thinking:

Reply:
also patched input for test so we can test the result is `(1)`